airbyte-integrations/connectors/destination-chroma/destination_chroma/config.py

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Literal, Union
+
+import dpath.util
+from airbyte_cdk.destinations.vector_db_based.config import (
+    CohereEmbeddingConfigModel,
+    FakeEmbeddingConfigModel,
+    FromFieldEmbeddingConfigModel,
+    OpenAIEmbeddingConfigModel,
+    ProcessingConfigModel,
+)
+from airbyte_cdk.utils.spec_schema_transformations import resolve_refs
+from pydantic import BaseModel, Field
+
+
+class HttpMode(BaseModel):
+    mode: Literal["http_client"] = Field("http_client", const=True)
+    host: str = Field(..., title="Host", description="The URL to the chromadb instance", order=0)
+    port: str = Field(..., title="Port", description="The port to the chromadb instance", order=1)
+    username: str = Field(..., title="Username", description="Username used in server/client mode only", order=2)
+    password: str = Field(..., title="Password", description="Password used in server/client mode only", airbyte_secret=True, order=3)
+
+    class Config:
+        title = "Client/Server mode"
+        schema_extra = {"description": "Authenticate using username and password (suitable for self-managed Chroma clusters)"}
+
+
+class PersistentMode(BaseModel):
+    mode: Literal["persistent_client"] = Field("persistent_client", const=True)
+    path: str = Field(..., title="Path", description="Where Chroma will store its database files on disk, and load them on start.")
+
+    class Config:
+        title = "Persistent Client"
+        schema_extra = {"description": "Configure Chroma to save and load from your local machine"}
+
+
+class ChromaIndexingConfigModel(BaseModel):
+
+    auth_method: Union[PersistentMode, HttpMode] = Field(
+        ..., title="Authentication", description="Authentication method", discriminator="mode", type="object", order=0
+    )
+    collection_name: str = Field(..., title="Collection Name", description="The collection to load data into", order=3)
+
+    class Config:
+        title = "Indexing"
+        schema_extra = {
+            "group": "indexing",
+            "description": "Indexing configuration",
+        }
+
+
+class NoEmbeddingConfigModel(BaseModel):
+    mode: Literal["no_embedding"] = Field("no_embedding", const=True)
+
+    class Config:
+        title = "Chroma Default Embedding Function"
+        schema_extra = {"description": "Do not calculate embeddings. Use Chromadb default embedding function"}
```

Comment:
Could you mention the model that's used here in the description? Also, depending on your hardware this can get very slow, could you mention this in the description?